#!/bin/bash

echo "diff --git a/Sources/Yams/CMakeLists.txt b/Sources/Yams/CMakeLists.txt
index 3e84614..1235fd0 100644
--- a/Sources/Yams/CMakeLists.txt
+++ b/Sources/Yams/CMakeLists.txt
@@ -31,8 +31,8 @@ set_target_properties(Yams PROPERTIES
 set_property(GLOBAL APPEND PROPERTY YAMS_EXPORTS Yams)
 get_swift_host_arch(swift_arch)
 install(FILES
-  \${CMAKE_CURRENT_BINARY_DIR}/swift/Yams.swiftmodule
-  \${CMAKE_CURRENT_BINARY_DIR}/swift/Yams.swiftdoc
+  \${CMAKE_BINARY_DIR}/swift/Yams.swiftmodule
+  \${CMAKE_BINARY_DIR}/swift/Yams.swiftdoc
   DESTINATION \${CMAKE_INSTALL_LIBDIR}/\${swift_arch})
 install(TARGETS Yams
   EXPORT YamsExports
" | patch -p1